app/main.py

```diff
@@ -1,13 +1,31 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        **{
+            k: error[v]
+            for k, v in {
+                "line": "line_number",
+                "column": "column_number",
+                "message": "text",
+                "name": "code"
+            }.items()
+        },
+        "source": "flake8"
```

Comment:
Why? You can simplify your code
```suggestion
    {
        "line": error["line_number"],
        "column": error["column_number"],
        "message": error["text"],
        "name": error["code"],
        "source": "flake8"
    }
```